docs/features/datastores/attaching-datastores.md

```diff
@@ -0,0 +1,81 @@
+
+# Attaching Datastores
+Datastores can be attached to both Runs and Sessions. Below are examples on how to do that.
+
+## Runs
+
+<!-- ![](/images/data.gif) -->
+
+You can mount a datastore to a run to make your experiments run faster! By default, the datastore is mounted at /datastores. When attaching datastores to a run, take note of the path your script uses. For example if your script takes an argument _my_data_path_ and you want to mount the _cats_ datastore:
+
+```bash
+grid run main.py --my_data_path /datastores/cats/1
```

Comment:
we should explain that the 1 means datastore version

Reply:
that "1" should be removed. I learned late last week that mounting datastores on sessions places them at `/datastores/<datastore-name>` if memory serves correct

Reply:
confirmation screenshot. That behavior change was never communicated or it is a regression.
![Screenshot 2022-03-27 222858](https://user-images.githubusercontent.com/47154698/160316330-cd11b16e-4d8e-4e82-915e-ead742af33bc.png)